cmd/guacone/cmd/files.go

```diff
@@ -126,17 +123,7 @@ var filesCmd = &cobra.Command{
 
 		gotErr := false
 
-		// Backend can only process a few files at a time. Increasing this might cause timeout errors in the database
-		maxConcurrentJobs, found := os.LookupEnv(maxConcurrentJobsString)
-		if found {
-			jobs, err := strconv.Atoi(maxConcurrentJobs)
-			if err != nil {
-				logger.Fatalf("failed to convert concurrent jobs value to integer ")
-			}
-			files.SetLimit(jobs)
-		} else {
-			files.SetLimit(1)
-		}
+		files.SetLimit(1)
```

Comment:
Can remove all this completely. `files, filesCtx := errgroup.WithContext(ctx)` and replace with the regular `ctx`